src/SystemPaths.cc

```diff
@@ -100,11 +117,23 @@ SystemPaths::SystemPaths()
     createDirectories(fullPath);
   }
 
-
   this->dataPtr->logPath = fullPath;
   // Populate this->dataPtr->filePaths with values from the default
   // environment variable.
-  this->SetFilePathEnv(this->dataPtr->filePathEnv);
+
+  if (this->dataPtr->filePathEnv.empty() &&
+      !this->dataPtr->filePathEnv.empty())
```

Comment:
Humm can this ever be true? I think some variables got mixed up here

Reply:
Oops

Reply:
https://github.com/gazebosim/gz-common/pull/365/commits/cb5f31e2958f927c0bcab1db07114fd24df82668